discovery-provider/alembic/trigger_sql/ddl.sql

```diff
@@ -135,3 +135,9 @@ BEGIN;
     END IF;
     END $$;
 COMMIT;
+
+-- 5/1/23: add AI attribution index
+
+BEGIN;
+    create index if not exists tracks_ai_attribution on tracks (ai_attribution, is_current);
```

Comment:
could be (ai_attribution_user_id, is_current=true)

Reply:
yea need this change

Reply:
good idea

Reply:
added. fwiw other indices don't have this though (maybe should add?)

```
    "tracks_track_cid_idx" btree (track_cid, is_current, is_delete)
    "track_is_premium_idx" btree (is_premium, is_current, is_delete)
```

Reply:
yeah probably, can save for a later date